js/interface.js

```diff
@@ -58,7 +59,16 @@ function getDataSources() {
   $dataSources = $('#data-sources > tbody');
 
   Fliplet.DataSources.get().then(function (dataSources) {
```

Comment:
That works.
But here you want to get `type: null` because "data" data sources have no type.